.travis/yapf.sh

```diff
@@ -9,17 +9,54 @@ builtin cd "$(dirname "${BASH_SOURCE:-$0}")"
 ROOT="$(git rev-parse --show-toplevel)"
 builtin cd "$ROOT"
 
-yapf \
-    --style "$ROOT/.style.yapf" \
-    --in-place --recursive --parallel \
-    --exclude 'python/ray/cloudpickle' \
-    --exclude 'python/ray/dataframe' \
-    --exclude 'python/ray/rllib' \
-    -- \
-    'test' 'python'
+YAPF_FLAGS="--style $ROOT/.style.yapf --in-place --recursive --parallel"
+
+YAPF_EXCLUDES="--exclude 'python/ray/cloudpickle' \
+--exclude 'python/ray/dataframe' \
+--exclude 'python/ray/rllib' \
+--exclude 'python/build' \
+--exclude 'python/ray/pyarrow_files' \
+--exclude 'python/ray/core/src/ray/gcs' \
+--exclude 'python/ray/common/thirdparty'"
+
+# Add the upstream branch if it doesn't exist
+if [ ! -e $ROOT/.git/refs/remotes/upstream ]; then
```

Comment:
@ericl FYI, you should use `[[` in bash over `[`, since `[[` can warn of syntax errors because it's part of the syntax rather than an external command.